src/replays/replayitem.py

```diff
@@ -164,12 +184,25 @@ def _on_map_preview_downloaded(self, mapname, result):
         self.icon = util.THEME.icon(path, is_local)
         self.setIcon(0, self.icon)
 
-    def infoPlayers(self, players):
+    def infoPlayers(self):
         """ processes information from the server about a replay into readable extra information for the user,
                 also calls method to show the information """
-
+        
         self.moreInfo = True
-        self.numberplayers = len(players)
+        playersList = self.replay['relationships']['playerStats']['data']
+        self.numberplayers = len(playersList)
+        
+        players = []
+        
+        for player in playersList:
+            #this is just SAD
```

Comment:
why?

Reply:
These arrays gave me headache :D II forgot to remove this comment. Also I think there was really crazy and scary stuff in this section at the begining, then I clear it  a bit :)